src/DependencyInjection/Providers/GoogleProviderConfigurator.php

```diff
@@ -19,10 +19,10 @@ public function buildConfiguration(NodeBuilder $node)
         // todo - add the comments as help text, and render in README
         $node
             ->scalarNode('access_type')
-                ->info('Optional value for sending hd parameter. More detail: https://developers.google.com/accounts/docs/OAuth2Login#hd-param')
+                ->info('Optional value for sending access_type parameter. More detail: https://developers.google.com/identity/protocols/OAuth2WebServer#offline')
```

Comment:
What about a link to the auth params instead?
https://developers.google.com/identity/protocols/OpenIDConnect#authenticationuriparameters